### ISSUE TITLE
Contacts, Events, Glossary links and footer icon links are missing visible focus indicator when tabbing

### ISSUE THREAD

**Issue by mcharg** [DOI-ONRR/onrr.gov-site#1971]
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/06325c33-a4d3-43a7-b34c-c35954919bc1)

So do the logo/home, social, and contacts links in the footer 

![Image](https://github.com/DOI-ONRR/onrr.gov-site/assets/115593715/1258c2b8-b9cb-43a1-a0c4-b29b2df8bf27)

**Comment by eelzi-ONRR** [comment]
With WCGA 2.2 focus indicators moves from AAA to A, so once WCAG 2.2 is approved we will want to bump this up in priority.

Buttons are also missing a visible focus.

**Comment by eelzi-ONRR** [comment]
These can be given the same white outline for focus that the logo/home links get in the header: 

![Image](https://github.com/DOI-ONRR/onrr.gov-site/assets/115593715/b3cb2b6e-4ae1-42c1-92a0-d88fb7d747e6)

**Comment by eelzi-ONRR** [comment]
The logo in the footer does have a focus but it doesn't work as required - it just shows a single line instead of enclosing the entire clickable area. The address next to it also get the line which should be removed since it's not a link/interactive area.

**Comment by greensomes** [comment]
A fix to the links mentioned is available in preview. I'm working on a fix for the footer logo. @eelzi-ONRR , do we need visible focus indicators for all links? If so, we'll need to come up with a different color or style for links that have white background.

**Comment by greensomes** [comment]
The links in the footer now have the same white outline for the tab focus.

**Comment by lpgoldstein** [comment]
@greensomes these are good to go! closing the issue
